calculator_test.go

```diff
@@ -2,23 +2,170 @@ package calculator_test
 
 import (
 	"calculator"
+	"math/rand"
 	"testing"
+	"time"
 )
 
 func TestAdd(t *testing.T) {
 	t.Parallel()
-	var want float64 = 4
-	got := calculator.Add(2, 2)
-	if want != got {
-		t.Errorf("want %f, got %f", want, got)
+	var testCases = []struct {
```

Comment:
And we could test Add, Subtract, and Multiply all in the same test, couldn't we? Can you see how?

Reply:
Addressed in https://github.com/micuffaro/ftl-fundamentals/pull/1/commits/2456ac63b78579d40850eaebb74ce1bf25b512d8